utils.go

```diff
@@ -61,3 +61,43 @@ func DecodeLabel(image imgutil.Image, label string, v interface{}) error {
 	}
 	return nil
 }
+
+// SyncLabels adds/updates/removes labels matching the predicate and a source image
+func SyncLabels(sourceImg imgutil.Image, destImage imgutil.Image, test func(string) bool) error {
```

Comment:
Is it possible to add unit tests for these functions?

Reply:
Agree, or alternatively, if we don't think these are likely to be reusable/useful library functions, we should unexport them.

Reply:
I went ahead and unexported them. I don't think they'll be used outside of the library, so I rather not export them at this time. The test I did add, I think, is probably enough testing given most of the heavy lifting lives in `imgutil`.